DevelopingSpace/starchart
# Update @remix-run dependencies

As per https://github.com/DevelopingSpace/starchart/pull/300#issuecomment-1460191483, this PR updates all `@remix-run` dependencies at once.

### Steps to test
- Run `npm install`
- Run `npm run build`
- Run `npm run docker`
- Run `npm run dev`
- Ensure all pages work as expected
